zio-k8s-operator/src/main/scala/com/coralogix/zio/k8s/operator/OperatorFailure.scala

```diff
@@ -46,8 +46,13 @@ object OperatorFailure {
       new RuntimeException(s"Not found")
   }
 
-  private def reqInfoToString(requestInfo: K8sRequestInfo): String =
-    s"[${requestInfo.resourceType.group}/${requestInfo.resourceType.version}/${requestInfo.resourceType.resourceType} ${requestInfo.operation}]"
+  private def reqInfoToString(requestInfo: K8sRequestInfo): String = {
+    val namespace = requestInfo.namespace.fold("")(n => s"namespace: $n")
+    val name = requestInfo.name.fold("")(n => s"name: $n")
+    val fieldSelector = requestInfo.fieldSelector.fold("")(n => s"field_selector: $n")
+    val labelSelector = requestInfo.labelSelector.fold("")(n => s"label_selector: $n")
+    s"[${requestInfo.resourceType.group}/${requestInfo.resourceType.version}/${requestInfo.resourceType.resourceType} ${requestInfo.operation} $namespace $name $fieldSelector $labelSelector]"
```

Comment:
I feel like this will make the errors a bit hard to read. As this is used to generate exception messages, it used to lead to something like:

```
[core/v1/Pod create] Failed to send request
```

Now it will be
```
[core/v1/Pod create Some(default) Some(xyz) None None] Failed to send request
```

What I think would be better:
- to move the extra information to the end
- hide if `None`, unwrap `Some`
- add info about what it is

So it would look like

```
[core/v1/Pod create] Failed to send request (namespace=default, name=xyz)
```

Reply:
Hey @vigoo thank you for your review.

I was already unwrapping the Options here:
```
    val namespace = requestInfo.namespace.fold("")(n => s"namespace=$n")
    val name = requestInfo.name.fold("")(n => s"name=$n")
    val fieldSelector = requestInfo.fieldSelector.fold("")(n => s"field_selector=$n")
    val labelSelector = requestInfo.labelSelector.fold("")(n => s"label_selector=$n")
```

I also moved the extra information to the end of the message. 
* Do you think field selector and label selector are useful in the message? Maybe it ends up being to large in some cases.
* The "error formatter" I've done is very naive (extra spaces if some Nones, etc). Imho it gives us enough value but I want to know your opinion too. :)

Reply:
I'm not sure, but we can remove it later if it gets too annoying :)